module/modmanager/module.go

```diff
@@ -310,7 +310,8 @@ func (m *module) stopProcess() error {
 	// SIGTERM correctly.
 	// Also ignore if error is that the process no longer exists.
 	if err := m.process.Stop(); err != nil {
-		if strings.Contains(err.Error(), errMessageExitStatus143) || strings.Contains(err.Error(), "no such process") {
+		var processNotExistsErr *pexec.ProcessNotExistsError
```

Comment:
It's our error not pexec's, right? You'll need to bump the `goutils` version, too.